charts/capsule-proxy/README.MD

```diff
@@ -0,0 +1,132 @@
+   
+        $ helm uninstall capsule-proxy -n capsule-system
+
+## Customize the installation
+
+There are two methods for specifying overrides of values during chart installation: `--values` and `--set`.
+
+The `--values` option is the preferred method because it allows you to keep your overrides in a YAML file, rather than specifying them all on the command line. Create a copy of the YAML file `values.yaml` and add your overrides to it.
+
+Specify your overrides file when you install the chart:
+
+        $ helm install capsule-proxy capsule-helm-chart --values myvalues.yaml -n capsule-system   
+
+The values in your overrides file `myvalues.yaml` will override their counterparts in the chart’s values.yaml file. Any values in `values.yaml` that weren’t overridden will keep their defaults.
+
+If you only need to make minor customizations, you can specify them on the command line by using the `--set` option. For example:
+
+        $ helm install capsule-proxy capsule-helm-chart --set force_tenant_prefix=false -n capsule-system
+
+Here the values you can override:
+
+Parameter | Description | Default
+--- | --- | ---
+`image.repository` | Set the image repository of the capsule-proxy. | `quay.io/clastix/capsule-proxy`
+`image.pullPolicy` | Set the image pull policy. | `IfNotPresent`
+`image.tag` | Overrides the image tag whose default is the chart. `appVersion` | `null`
+`options.logLevel` | Set the log verbosity of the capsule-proxy with a value from 1 to 10.| `5`
+`options.k8sControlPlaneUrl` | Set the URL of kubernetes control plane. | `https://kubernetes.default.svc`
+`options.capsuleUserGroup` | Override the Capsule user group | `capsule.clastix.io`
+`options.oidcUsernameClaim` | Override the OIDC field name used to identify the user | `preferred_username`
+`options.enableSSL` | Specify if capsule-proxy will use SSL | `false`
+`options.SSLDirectory` | Set the directory, where SSL certificate and keyfile will be located | `/opt/certs`
+`options.SSLCertFileName` | Set the name of SSL certificate file | `tls.crt`
+`options.SSLKeyFileName` | Set the name of SSL key file | `tls.key`
+`options.generateCertificates` | Specify if capsule-proxy will generate self-signed SSL certificates | `false`
+`imagePullSecrets` | Configuration for `imagePullSecrets` so that you can use a private images registry. | `[]`
+`serviceAccount.create` | Specifies whether a service account should be created. | `true`
+`serviceAccount.annotations` | Annotations to add to the service account. | `{}`
+`serviceAccount.name` | The name of the service account to use. If not set and `serviceAccount.create=true`, a name is generated using the fullname template | `capsule-proxy`
+`podAnnotations` | Annotations to add to the capsule-proxy pod. | `{}`
+`podSecurityContext` | Security context for the capsule-proxy pod. | `{}`
+`securityContext` | Security context for the capsule-proxy deployment. | `{}`
+`ingress.enabled` | Specifies whether an ingress should be created. | `true`
```

Comment:
Default is reported as `true` but in `values.yaml` file the default is `false`

Reply:
fixed